core/transactions/transactions_config.cxx

```diff
@@ -25,8 +25,7 @@ namespace couchbase::transactions
 {
 
 transactions_config::transactions_config()
-  : level_(couchbase::durability_level::majority)
-  , timeout_(std::chrono::seconds(15))
+  : timeout_(std::chrono::seconds(15))
```

Comment:
Should this also be removed, similar to `level_`? It's being initialized to 15 seconds by default.

```c++
  std::chrono::nanoseconds timeout_{ std::chrono::seconds(15) };
```